event-adapter/adapter.yaml

```diff
@@ -0,0 +1,134 @@
+# TODO(kawych): fix namespaces
```

Comment:
Please be more specific than "fix". What should be done wrt. namespaces? Will Karol do that?